test/integration/mediatorAPITests.coffee

```diff
@@ -35,7 +35,7 @@ describe "API Integration Tests", ->
             type: 'http'
           }
         ]
-      ,
+      }, {
```

Comment:
These aren't necessary because of coffee script. Where you noticing issues with them?